### ISSUE TITLE
[A11Y] Improve content and accessibility

### ISSUE THREAD

**Issue by shrinkray** [GDG-Detroit/devfest-website#79]
## Accessibility Issue Description

A number of issues have been brought to my attention which I introduced. In the results listed below there are a number of color contrast issues viewed as serious. 

The other accessibility issue has to do with the `aria-label` & `aria-description` on the maps. Both descriptions show 'map' and should instead be useful and informative. 

<img width="791" height="411" alt="Image" src="https://github.com/user-attachments/assets/38d612b6-6b54-41a2-927e-40a8c2f0c971" />

## WCAG Guidelines Violated

- [x] Perceivable (1.x)
- [ ] Operable (2.x)
- [ ] Understandable (3.x)
- [X] Robust (4.x)

## Specific WCAG Criteria

- [ ] 1.1.1 Non-text Content
- [ ] 1.3.1 Info and Relationships
- [X] 1.4.3 Contrast (Minimum)
- [ ] 1.4.4 Resize text
- [ ] 2.1.1 Keyboard
- [ ] 2.1.2 No Keyboard Trap
- [ ] 2.4.1 Bypass Blocks
- [ ] 2.4.2 Page Titled
- [ ] 2.4.3 Focus Order
- [ ] 2.4.4 Link Purpose
- [ ] 3.1.1 Language of Page
- [X] 4.1.2 Name, Role, Value
- [ ] Other (please specify):

## Steps to Reproduce

AXE utility in DevTools

## Expected Behavior

- Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
- Expected contrast ratio of 4.5:1
- Ensure landmarks are unique
- The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

## Actual Behavior

It's not a behavior issue its about the content

## Assistive Technology Used

- [ ] Screen reader (NVDA, JAWS, VoiceOver, etc.)
- [ ] Keyboard only navigation
- [ ] Voice control
- [ ] High contrast mode
- [ ] Zoom/magnification
- [x] Other (please specify): AXE utility

## Impact Level

- [ ] Critical - Prevents access to content/functionality
- [x] High - Significantly impacts user experience
- [ ] Medium - Somewhat impacts user experience
- [ ] Low - Minor impact on user experience

## Suggested Solution

- Increase contrast. either make the yellow ribbon darker or text darker and ribbon lighter
- Reword the `aria-label` and `aria-description` to provide useful info about the maps they are associated to

## Additional Context

Description subheadings should be added to each of the heading tags for H2  and H3 sections. It would be useful to all who visit, and it can be short, much like what was added to the Teams section. 

## Testing Environment

- **Browser**: Polypane
- **Assistive Technology**:
- **Operating System**: MacOS, Android

**Comment by shrinkray** [comment]
**Issue with mobile nav — horizontal overflow detected!**

<img width="286" height="547" alt="Image" src="https://github.com/user-attachments/assets/d87bde08-98da-467e-9084-a8c870c74898" />

<img width="289" height="538" alt="Image" src="https://github.com/user-attachments/assets/a4e98a83-6d2a-4bb5-99cc-63eb26f96a45" />

<img width="283" height="536" alt="Image" src="https://github.com/user-attachments/assets/c5b581d7-a53a-4cbe-8bc4-74da007810b0" />
